tests/queries/0_stateless/02420_mt_ordinary_select_final_setting.sql

```diff
@@ -0,0 +1 @@
+create table if not exists ordinary_mt (x String) engine=MergeTree() ORDER BY x SETTINGS force_select_final=1; -- { serverError 181 }
```

Comment:
Why does every test sit in a separate file? Especially for single-line 'unhappy path' scenario - having a separate test looks bad... 

Usually, features/cases are grouped by feature / single logical unit, a single test file can cover several scenarios by a single test file.

Also it would be nice to add the exception text in the comment to help test readers understand why it should fail.

Reply:
Hmmm, I simply thought the standard was to have 1 test per file mainly because of the expectation file. In such error tests, I think it can be incorporated into the other ones. Should even "happy" tests be merged? If so, what's the standard for expectation files? Echo to differentiate queries?

Reply:
Yep, echo is nice.